dev-aws/kafka-shared/customer-billing.tf

```diff
@@ -29,6 +29,15 @@ module "bills_total_api_consumer" {
   cert_common_name = "customer-billing/bills-total-api"
 }
 
+module "billing_fulfilment_public_events_translator_consumer" {
```

Comment:
Hey @JDGarner . I got a PR in flight that will refactor a bit this file, as we're moving away from the producer/consumer modules in favor of a new tls-app one: https://github.com/utilitywarehouse/kafka-cluster-config/pull/120. 

Can you please addapt this one too and wayt until I merge the other one, please

Reply:
Sure no problem 👍

Reply:
PRs are merged, you can go ahead with adding this.
Please contact me if you need help in adjusting these changes. Hopefully it's straight forwards